### PR TITLE
PMM-2301 Disable anonymous access.

### PR DESCRIPTION
**Pull request by AlekSi**
https://github.com/percona/pmm-managed/pull/234

See also https://github.com/Percona-Lab/percona-images/pull/31.

### CODE REVIEW

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-354-ff3588d
client docker - perconalab/pmm-client-fb:PR-354-ff3588d
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-354-ff3588d.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-354-71a2a38
client docker - perconalab/pmm-client-fb:PR-354-71a2a38
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-354-71a2a38.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-354-aadea75
client docker - perconalab/pmm-client-fb:PR-354-aadea75
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-354-aadea75.tar.gz

**Comment by AlekSi** [comment]
@hors PTAL

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-354-b15db2e
client docker - perconalab/pmm-client-fb:PR-354-b15db2e
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-354-b15db2e.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-354-42df868
client docker - perconalab/pmm-client-fb:PR-354-42df868
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-354-42df868.tar.gz

File: build/pmm-server-docker/playbook-install.yml [hunk 1]
```diff
@@ -280,37 +280,21 @@
...
-        value: false
-
-    - name: Grafana                    | Disable Anonymous access
-      when: not grafana_db.stat.exists
-      ini_file:
-        dest: /etc/grafana/grafana.ini
-        section: auth.anonymous
-        option: org_role
-        value: Editor
+        value: 'false'
```
Comment (hors) [inline build/pmm-server-docker/playbook-install.yml]: Why do you quote it?

Comment (AlekSi) [reply build/pmm-server-docker/playbook-install.yml]:
```
[WARNING]: The value False (type bool) in a string field was converted to
u'False' (type string). If this does not look like what you expect, quote the
entire value to ensure it does not change.
```

Comment (AlekSi) [reply build/pmm-server-docker/playbook-install.yml]: We also use 'false' and 'true' in Grafana config, not 'False' and 'True'.

**Pull request merged**